libs/core/src/lib/slider/slider.component.html

```diff
@@ -67,6 +67,7 @@
         [closeOnOutsideClick]="false"
         [appendTo]="handle"
         [disabled]="!tooltipMode"
+        [additionalBodyClass]="'fd-slider--tooltip-popover'"
```

Comment:
unnecessary binding

Reply:
It is necessary to prevent styles from slider's popover to interfere with other popover-like elements